test/plugin/scenarios/rocketmq-scenario/support-version.list

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# lists your version here (Contains only the last version number of each minor version.)
+
+4.9.4
+4.8.0
+4.7.1
+4.6.0
+4.5.2
+4.4.0
+4.3.1
```

Comment:
So, 4.2, 4.1, and 4.0 are not supported? Do I miss anything?

Reply:
These are not mentioned in [official docker hub](https://hub.docker.com/r/apache/rocketmq/tags?page=1).

Reply:
I see. Make sense.